src/Wizard.php

```diff
@@ -62,7 +62,7 @@ public function init(): void
         parent::init();
         $this->stepCallback = Callback::addTo($this, ['urlTrigger'=>$this->name]);
 
-        $this->currentStep = $this->stepCallback->triggered() ?: 0;
+        $this->currentStep = (int) $this->stepCallback->triggered() ?: 0;
```

Comment:
@ibelar#4582, @DarkSide666#2795 so this https://github.com/atk4/ui/blob/06fc859685cdf5f9f3d19772f4305e9eac2d99c0/src/Callback.php#L140 should return string then?

Or introduce triggeredStr() and triggered()? Or propose a better alternative.